### ISSUE TITLE
2024-11-27, Daily Papers for llm, large language model

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#1]
# Daily Arxiv Papers, 2024-11-27

(1) [Adaptive Deployment of Untrusted LLMs Reduces Distributed Threats](http://arxiv.org/abs/2411.17693v1), 2024-11-26

(2) [Low-Bit Quantization Favors Undertrained LLMs: Scaling Laws for Quantized LLMs with 100T Training Tokens](http://arxiv.org/abs/2411.17691v1), 2024-11-26

(3) [Rethinking Token Reduction in MLLMs: Towards a Unified Paradigm for Training-Free Acceleration](http://arxiv.org/abs/2411.17686v1), 2024-11-26

(4) [Enhancing Character-Level Understanding in LLMs through Token Internal Structure Learning](http://arxiv.org/abs/2411.17679v1), 2024-11-26

(5) [Push the Limit of Multi-modal Emotion Recognition by Prompting LLMs with Receptive-Field-Aware Attention Weighting](http://arxiv.org/abs/2411.17674v1), 2024-11-26

(6) [SketchAgent: Language-Driven Sequential Sketch Generation](http://arxiv.org/abs/2411.17673v1), 2024-11-26

(7) [Synthetic Data Generation with LLM for Improved Depression Prediction](http://arxiv.org/abs/2411.17672v1), 2024-11-26

(8) [Toward High-Performance LLM Serving: A Simulation-Based Approach for Identifying Optimal Parallelism](http://arxiv.org/abs/2411.17651v1), 2024-11-26

(9) [On Limitations of LLM as Annotator for Low Resource Languages](http://arxiv.org/abs/2411.17637v1), 2024-11-26

(10) [MALMM: Multi-Agent Large Language Models for Zero-Shot Robotics Manipulation](http://arxiv.org/abs/2411.17636v1), 2024-11-26

(11) [Data-driven development of cycle prediction models for lithium metal batteries using multi modal mining](http://arxiv.org/abs/2411.17625v1), 2024-11-26

(12) [Scaling Speech-Text Pre-training with Synthetic Interleaved Data](http://arxiv.org/abs/2411.17607v1), 2024-11-26

(13) [HyperSeg: Towards Universal Visual Segmentation with Large Language Model](http://arxiv.org/abs/2411.17606v1), 2024-11-26

(14) [Making History Readable](http://arxiv.org/abs/2411.17600v1), 2024-11-26

(15) [Agentic AI for Improving Precision in Identifying Contributions to Sustainable Development Goals](http://arxiv.org/abs/2411.17598v1), 2024-11-26

(16) [Can artificial intelligence predict clinical trial outcomes?](http://arxiv.org/abs/2411.17595v1), 2024-11-26

(17) [RTL-Breaker: Assessing the Security of LLMs against Backdoor Attacks on HDL Code Generation](http://arxiv.org/abs/2411.17569v1), 2024-11-26

(18) [Natural Language Understanding and Inference with MLLM in Visual Question Answering: A Survey](http://arxiv.org/abs/2411.17558v1), 2024-11-26

(19) [Pushing the Limits of Large Language Model Quantization via the Linearity Theorem](http://arxiv.org/abs/2411.17525v1), 2024-11-26

(20) [Inference Scaling $\scriptsize\mathtt{F}$Laws: The Limits of LLM Resampling with Imperfect Verifiers](http://arxiv.org/abs/2411.17501v1), 2024-11-26

(21) [SoK: Decentralized AI (DeAI)](http://arxiv.org/abs/2411.17461v1), 2024-11-26

(22) [PEFTGuard: Detecting Backdoor Attacks Against Parameter-Efficient Fine-Tuning](http://arxiv.org/abs/2411.17453v1), 2024-11-26

(23) ["Stupid robot, I want to speak to a human!" User Frustration Detection in Task-Oriented Dialog Systems](http://arxiv.org/abs/2411.17437v1), 2024-11-26

(24) [BPP-Search: Enhancing Tree of Thought Reasoning for Mathematical Modeling Problem Solving](http://arxiv.org/abs/2411.17404v1), 2024-11-26

(25) [One Mind, Many Tongues: A Deep Dive into Language-Agnostic Knowledge Neurons in Large Language Models](http://arxiv.org/abs/2411.17401v1), 2024-11-26

(26) [Can LLMs be Good Graph Judger for Knowledge Graph Construction?](http://arxiv.org/abs/2411.17388v1), 2024-11-26

(27) [The Extractive-Abstractive Spectrum: Uncovering Verifiability Trade-offs in LLM Generations](http://arxiv.org/abs/2411.17375v1), 2024-11-26

(28) [Different Bias Under Different Criteria: Assessing Bias in LLMs with a Fact-Based Approach](http://arxiv.org/abs/2411.17338v1), 2024-11-26

(29) [MotionLLaMA: A Unified Framework for Motion Synthesis and Comprehension](http://arxiv.org/abs/2411.17335v1), 2024-11-26

(30) [InsightEdit: Towards Better Instruction Following for Image Editing](http://arxiv.org/abs/2411.17323v1), 2024-11-26

(31) [PIM-AI: A Novel Architecture for High-Efficiency LLM Inference](http://arxiv.org/abs/2411.17309v1), 2024-11-26

(32) [Meaningless is better: hashing bias-inducing words in LLM prompts improves performance in logical reasoning and statistical learning](http://arxiv.org/abs/2411.17304v1), 2024-11-26

(33) [ER2Score: LLM-based Explainable and Customizable Metric for Assessing Radiology Reports with Reward-Control Loss](http://arxiv.org/abs/2411.17301v1), 2024-11-26

(34) [Using Large Language Models for Expert Prior Elicitation in Predictive Modelling](http://arxiv.org/abs/2411.17284v1), 2024-11-26

(35) [CleanVul: Automatic Function-Level Vulnerability Detection in Code Commits Using LLM Heuristics](http://arxiv.org/abs/2411.17274v1), 2024-11-26

(36) [A Topic-level Self-Correctional Approach to Mitigate Hallucinations in MLLMs](http://arxiv.org/abs/2411.17265v1), 2024-11-26

(37) [HEIE: MLLM-Based Hierarchical Explainable AIGC Image Implausibility Evaluator](http://arxiv.org/abs/2411.17261v1), 2024-11-26

(38) [APT: Architectural Planning and Text-to-Blueprint Construction Using Large Language Models for Open-World Agents](http://arxiv.org/abs/2411.17255v1), 2024-11-26

(39) [Grounding-IQA: Multimodal Language Grounding Model for Image Quality Assessment](http://arxiv.org/abs/2411.17237v1), 2024-11-26

(40) [Fault Localization from the Semantic Code Search Perspective](http://arxiv.org/abs/2411.17230v1), 2024-11-26

(41) [Strategic Prompting for Conversational Tasks: A Comparative Analysis of Large Language Models Across Diverse Conversational Tasks](http://arxiv.org/abs/2411.17204v1), 2024-11-26

(42) [Exploring the Impact of Anthropomorphism in Role-Playing AI Chatbots on Media Dependency: A Case Study of Xuanhe AI](http://arxiv.org/abs/2411.17157v1), 2024-11-26

(43) [LLM-Based Offline Learning for Embodied Agents via Consistency-Guided Reward Ensemble](http://arxiv.org/abs/2411.17135v1), 2024-11-26

(44) [TechCoach: Towards Technical Keypoint-Aware Descriptive Action Coaching](http://arxiv.org/abs/2411.17130v1), 2024-11-26

(45) [DOGE: Towards Versatile Visual Document Grounding and Referring](http://arxiv.org/abs/2411.17125v1), 2024-11-26

(46) [Advancing Content Moderation: Evaluating Large Language Models for Detecting Sensitive Content Across Text, Images, and Videos](http://arxiv.org/abs/2411.17123v1), 2024-11-26

(47) [Star Attention: Efficient LLM Inference over Long Sequences](http://arxiv.org/abs/2411.17116v1), 2024-11-26

(48) [TabulaX: Leveraging Large Language Models for Multi-Class Table Transformations](http://arxiv.org/abs/2411.17110v1), 2024-11-26

(49) [Scholar Name Disambiguation with Search-enhanced LLM Across Language](http://arxiv.org/abs/2411.17102v1), 2024-11-26

(50) [Efficient LLM Inference with I/O-Aware Partial KV Cache Recomputation](http://arxiv.org/abs/2411.17089v1), 2024-11-26

(51) [Creative Agents: Simulating the Systems Model of Creativity with Generative Agents](http://arxiv.org/abs/2411.17065v1), 2024-11-26

(52) [ThreatModeling-LLM: Automating Threat Modeling using Large Language Models for Banking System](http://arxiv.org/abs/2411.17058v1), 2024-11-26

(53) [Redefining Crowdsourced Test Report Prioritization: An Innovative Approach with Large Language Model](http://arxiv.org/abs/2411.17045v1), 2024-11-26

(54) [TED-VITON: Transformer-Empowered Diffusion Models for Virtual Try-On](http://arxiv.org/abs/2411.17017v1), 2024-11-26

(55) [Trinity: Synchronizing Verbal, Nonverbal, and Visual Channels to Support Academic Oral Presentation Delivery](http://arxiv.org/abs/2411.17015v1), 2024-11-26

(56) [Dynamic Self-Distillation via Previous Mini-batches for Fine-tuning Small Language Models](http://arxiv.org/abs/2411.16991v1), 2024-11-25

(57) [Enabling Skip Graphs to Process K-Dimensional Range Queries in a Mobile Sensor Network](http://dx.doi.org/10.1109/NCA.2007.18), 2024-11-25

(58) [Teaching Smaller Language Models To Generalise To Unseen Compositional Questions (Full Thesis)](http://arxiv.org/abs/2411.16985v1), 2024-11-25

(59) [Harnessing LLMs for Educational Content-Driven Italian Crossword Generation](http://arxiv.org/abs/2411.16936v1), 2024-11-25

(60) [Seq2Time: Sequential Knowledge Transfer for Video LLM Temporal Grounding](http://arxiv.org/abs/2411.16932v1), 2024-11-25

(61) [ASSERTIFY: Utilizing Large Language Models to Generate Assertions for Production Code](http://arxiv.org/abs/2411.16927v1), 2024-11-25

(62) [Augmenting Multimodal LLMs with Self-Reflective Tokens for Knowledge-based Visual Question Answering](http://arxiv.org/abs/2411.16863v1), 2024-11-25

(63) [SAR3D: Autoregressive 3D Object Generation and Understanding via Multi-scale 3D VQVAE](http://arxiv.org/abs/2411.16856v1), 2024-11-25

(64) [Do Large Language Models Perform Latent Multi-Hop Reasoning without Exploiting Shortcuts?](http://arxiv.org/abs/2411.16679v1), 2024-11-25

(65) [DreamRunner: Fine-Grained Storytelling Video Generation with Retrieval-Augmented Motion Adaptation](http://arxiv.org/abs/2411.16657v1), 2024-11-25

(66) [Beyond Sight: Towards Cognitive Alignment in LVLM via Enriched Visual Knowledge](http://arxiv.org/abs/2411.16824v1), 2024-11-25

(67) [Self-Generated Critiques Boost Reward Modeling for Language Models](http://arxiv.org/abs/2411.16646v1), 2024-11-25

(68) [Preventing Jailbreak Prompts as Malicious Tools for Cybercriminals: A Cyber Defense Perspective](http://arxiv.org/abs/2411.16642v1), 2024-11-25

(69) [Do Automatic Factuality Metrics Measure Factuality? A Critical Evaluation](http://arxiv.org/abs/2411.16638v2), 2024-11-25

(70) [Chat2SVG: Vector Graphics Generation with Large Language Models and Image Diffusion Models](http://arxiv.org/abs/2411.16602v1), 2024-11-25

(71) [From Generation to Judgment: Opportunities and Challenges of LLM-as-a-judge](http://arxiv.org/abs/2411.16594v1), 2024-11-25

(72) [Large Language Model-based Decision-making for COLREGs and the Control of Autonomous Surface Vehicles](http://arxiv.org/abs/2411.16587v1), 2024-11-25

(73) [MarketGPT: Developing a Pre-trained transformer (GPT) for Modeling Financial Time Series](http://arxiv.org/abs/2411.16585v1), 2024-11-25

(74) [Enhancing LLM Reasoning via Critique Models with Test-Time and Training-Time Supervision](http://arxiv.org/abs/2411.16579v1), 2024-11-25

(75) [Predictive Power of LLMs in Financial Markets](http://arxiv.org/abs/2411.16569v1), 2024-11-25

(76) [EnStack: An Ensemble Stacking Framework of Large Language Models for Enhanced Vulnerability Detection in Source Code](http://arxiv.org/abs/2411.16561v1), 2024-11-25

(77) [Generating Out-Of-Distribution Scenarios Using Language Models](http://arxiv.org/abs/2411.16554v1), 2024-11-25

(78) [Enhancing In-Hospital Mortality Prediction Using Multi-Representational Learning with LLM-Generated Expert Summaries](http://arxiv.org/abs/2411.16818v1), 2024-11-25

(79) [Representation Collapsing Problems in Vector Quantization](http://arxiv.org/abs/2411.16550v1), 2024-11-25

(80) [Profiling Bias in LLMs: Stereotype Dimensions in Contextual Word Embeddings](http://arxiv.org/abs/2411.16527v1), 2024-11-25

(81) [LaB-RAG: Label Boosted Retrieval Augmented Generation for Radiology Report Generation](http://arxiv.org/abs/2411.16523v1), 2024-11-25

(82) [Interpreting Language Reward Models via Contrastive Explanations](http://arxiv.org/abs/2411.16502v1), 2024-11-25

(83) [AtomR: Atomic Operator-Empowered Large Language Models for Heterogeneous Knowledge Reasoning](http://arxiv.org/abs/2411.16495v1), 2024-11-25

(84) [Fine-Tuning LLMs with Noisy Data for Political Argument Generation](http://arxiv.org/abs/2411.16813v1), 2024-11-25

(85) [Learning by Analogy: Enhancing Few-Shot Prompting for Math Word Problem Solving with Computational Graph-Based Retrieval](http://arxiv.org/abs/2411.16454v1), 2024-11-25

(86) [Blockchain Meets LLMs: A Living Survey on Bidirectional Integration](http://arxiv.org/abs/2411.16809v1), 2024-11-25

(87) [TopV-Nav: Unlocking the Top-View Spatial Reasoning Potential of MLLM for Zero-shot Object Navigation](http://arxiv.org/abs/2411.16425v1), 2024-11-25

(88) [A Multi-agent Framework for Materials Laws Discovery](http://arxiv.org/abs/2411.16416v1), 2024-11-25

(89) [FineWeb-zhtw: Scalable Curation of Traditional Chinese Text Data from the Web](http://arxiv.org/abs/2411.16387v1), 2024-11-25

(90) [The Two-Hop Curse: LLMs trained on A->B, B->C fail to learn A-->C](http://arxiv.org/abs/2411.16353v1), 2024-11-25

(91) [Towards Foundation Models for Critical Care Time Series](http://arxiv.org/abs/2411.16346v1), 2024-11-25

(92) [Preference Optimization for Reasoning with Pseudo Feedback](http://arxiv.org/abs/2411.16345v1), 2024-11-25

(93) [From CISC to RISC: language-model guided assembly transpilation](http://arxiv.org/abs/2411.16341v1), 2024-11-25

(94) [Can AI grade your essays? A comparative analysis of large language models and teacher ratings in multidimensional essay scoring](http://arxiv.org/abs/2411.16337v1), 2024-11-25

(95) [CATP-LLM: Empowering Large Language Models for Cost-Aware Tool Planning](http://arxiv.org/abs/2411.16313v1), 2024-11-25

(96) [BayLing 2: A Multilingual Large Language Model with Efficient Language Alignment](http://arxiv.org/abs/2411.16300v1), 2024-11-25

(97) [Unraveling Arithmetic in Large Language Models: The Role of Algebraic Structures](http://arxiv.org/abs/2411.16260v1), 2024-11-25

(98) [Enhancing Answer Reliability Through Inter-Model Consensus of Large Language Models](http://arxiv.org/abs/2411.16797v1), 2024-11-25

(99) [NormXLogit: The Head-on-Top Never Lies](http://arxiv.org/abs/2411.16252v1), 2024-11-25

(100) [CS-Eval: A Comprehensive Large Language Model Benchmark for CyberSecurity](http://arxiv.org/abs/2411.16239v1), 2024-11-25
